### PR TITLE
Move from maven to sbt

### PR DESCRIPTION
**Pull request by pedromdias**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  .aggregate(core)
+
+lazy val core = project
+  .in(file("core"))
+  .settings(
+    name := "akka-persistent-scheduler",
+    libraryDependencies ++= Seq(
+        "com.typesafe.akka"          %% "akka-testkit"                        % "2.4.17",
+        "org.postgresql"             % "postgresql"                           % "42.0.0",
+        "com.miguno.akka"            %% "akka-mock-scheduler"                 % "0.5.1",
```
Comment (dpfeiffer) [inline build.sbt]: This is a compile and not a test dependency?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#  script:
+#    - sbt publish
+#  on:
+#    tags: true
+#    repo: firstbirdtech/akka-persistent-scheduler
+
+env:
+  global:
+    # encrypt with: travis encrypt BINTRAY_USER=...
+    - secure: "WXITvMAQmjJHQijqCkS+cZ9BLEH+B9n3IiGtP3bV8U+9UsR4kU7+mZyDAQDdI6l36rHDgPPWqy/PCxnXETKY0do2RqI70lk3bgqmPUy7V6BP+pweX103fBuztEuPBm124nnUrJj7enRGDe+kwSoXCCwkidw3RjrJy2yNJBLgOSu7HPDfOFqnzsHiA/C1aithTCG0V7e0l6Lvb8HzXRg91pTAJGJQIm3Mff4mACpThyL90hfde+Jb2Is1KnpXTMCoGWC65wxbh8rjCmPSFs+98XeceAFDJqXiVKx/AWJDVMv11NYxOGqBDjC2w6LgBSvI140xm5Y7LhgeM5nZNiUaximoxAbmHFrddVOyxZ9jlNe+OLxo3FQXaa6+6MtXifZDbL3ikz/Ztyt0AE79o1Ui/GFLs2pKc2g1WlkelMH5qLmXt00lKzn1f/NN3k18WaBhQlKPosrEH0mKWlfyrkPETHi7e2fCh9UrKhDZw3/SjdFGMLMZfQr3JJCSBxusfngBMVQZ1ZJ3Fpzn2HteL1XLrssPUk+S9s+fNALW0UuaUu7rnEyjFCSnDoraTpjQboZXbpWz5qI/pIa5xcSv53jblYlwvRJbgsjc2LO1k/mHrZfJ8jVoo9vL04V0ch8kCCATeqaocdg2Wlb3fj0j5/hVwrDYJffoVlaHo2lB99yO+ds="
```
Comment (dpfeiffer) [inline .travis.yml]: Have you created new ones?

File: project/build.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+sbt.version = 0.13.13
```
Comment (dpfeiffer) [inline project/build.properties]: There is already 0.13.15

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+version: '2'
```
Comment (dpfeiffer) [inline docker-compose.yml]: Why do we need the docker-comose file?

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+   - 2.11.8
+
+jdk:
+   - oraclejdk8
+
+services:
+  - docker
+
+before_install:
+  - docker-compose up -d
```
Comment (dpfeiffer) [inline .travis.yml]: Why is that needed?

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+sudo: required
+
+scala:
+   - 2.11.8
+
+jdk:
+   - oraclejdk8
+
+services:
+  - docker
```
Comment (dpfeiffer) [inline .travis.yml]: If we don't use docker-compose we don't need that service

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+before_install:
+  - docker-compose up -d
+
+script:
+  - sbt clean coverage test coverageReport coverageAggregate
```
Comment (dpfeiffer) [inline .travis.yml]: We are doing nothing with the coverage. I propose to remove all coverage things for now and add them later again in a second PR (where we can also configure codacy properly)

File: .travis.yml (hunk 2 above)
Comment (pedromdias) [inline .travis.yml]: Do I need to? Why not use the same user, if we will all be using the same creds?

File: build.sbt [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  .in(file("core"))
+  .settings(
+    name := "akka-persistent-scheduler",
+    libraryDependencies ++= Seq(
+        "com.typesafe.akka"          %% "akka-stream"                         % "2.4.17",
+        "org.postgresql"             % "postgresql"                           % "42.0.0",
+        "org.scala-lang.modules"     %% "scala-java8-compat"                  % "0.8.0",
+        "joda-time"                  % "joda-time"                            % "2.9.4",
+        "org.joda"                   % "joda-convert"                         % "1.8.1",
+        "com.typesafe.akka"          %% "akka-testkit"                        % "2.4.17",
```
Comment (dpfeiffer) [inline build.sbt]: Testkit needs to be test dependency as well
